Trappist/src/Promact.Trappist.Web/webpack/resolve.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Use of JavaScript strict mode may result in unexpected behaviour in some browsers. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://server12:9005/coding_rules#rule_key=javascript%3AStrictMode)